packages/sidenav/sidenav-item.md

```diff
@@ -0,0 +1,62 @@
+## Description
+
+An `<sp-sidenav-item>` stands as both a child item of an `<sp-sidenav>` element as well as a parent for further subdivisions of that navigation. An `<sp-sidenav-item>` with further `<sp-sidenav-item>` children will count as a toggle for the visibility of this never level of navigation items, while also updating the parent `<sp-sidenav>` element to its value when selected.
```

Comment:
```suggestion
An `<sp-sidenav-item>` stands as both a child item of an `<sp-sidenav>` element, as well as a parent for further subdivisions of that navigation. An `<sp-sidenav-item>` with further `<sp-sidenav-item>` children will count as a toggle for the visibility of this never level of navigation items, while also updating the parent `<sp-sidenav>` element to its value when selected.
```
What word is "never" supposed to be? I feel that's a typo but I'm not sure what it should be

Reply:
"next"... 🙈

Reply:
```suggestion
An `<sp-sidenav-item>` stands as both a child item of an `<sp-sidenav>` element as well as a parent for further subdivisions of that navigation. An `<sp-sidenav-item>` with further `<sp-sidenav-item>` children will count as a toggle for the visibility of this next level of navigation items, while also updating the parent `<sp-sidenav>` element to its value when selected.
```